exercises/zebra-puzzle/README.md

```diff
@@ -27,20 +27,25 @@ Who owns the zebra?
 
 ## Implementation
 
-Define a single function, Solve, which returns two strings,
-whose values are the answers to the zebra-puzzle questions
-"Who drinks water?" and "Who owns the Zebra?".
-Each return value will be one of the resident's nationalities:
+Define a single function, SolvePuzzle, which returns a solution
```

Comment:
Since configlet generates readme automatically, how are we going to keep updates ?

Reply:
I generated this README.md using configlet, and it made use of the .meta/hints.md.
I thought that we checked in the generated README.md as well.  Do we not ?

Reply:
Sorry I was double checking my understanding.